azurerm/internal/services/netapp/data_source_netapp_volume.go

```diff
@@ -42,6 +42,19 @@ func dataSourceArmNetAppVolume() *schema.Resource {
 				ValidateFunc: ValidateNetAppPoolName,
 			},
 
+			"mount_targets": {
+				Type:     schema.TypeSet,
```

Comment:
this needs to be a TypeList if it's a Computed field else it's not easily indexable - however - what are the mount targets here, a list of IP Addresses which should be used to mount the volume? If so it's likely this'd make more sense as a list of strings being `mount_ip_addresses`?

can we also update the test to confirm this field is correctly set

Reply:
Fixed